cypress/integration/connect-wallet/test_connect_with_metamask.js

```diff
@@ -0,0 +1,21 @@
+describe('connect wallet: metamask', () => {
+  it('pulls address from metamask accounts', () => {
+    cy.visit('http://localhost:8000/_administrationlogin');
+
+    cy.get('[name=username]').type('root');
+    cy.get('[name=password]').type('gitcoinco');
+    cy.contains('Log in').click();
+
+    cy.contains('Impersonate Users').click();
+    cy.contains('test3').click();
```

Comment:
can these lines be made d.r.y so they don't repeat in every test?

Reply:
Yes. In a new branch I have started moving things to use Cypress commands to make this more reusable. I was doing more exploration here than anything else and didn't want to extract too soon. I like to follow Sandi Metz advice on DRY - https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction.